multi-repo-action/org-workflow-add.go

```diff
@@ -44,35 +63,72 @@ func main() {
 	err_check(err, "Error reading in scorecard workflow file.")
 
 	// Process each repository.
-	for _, repoName := range REPO_LIST {
-
+	for _, repoName := range RepoList {
 		// Get repo metadata.
 		repo, _, err := client.Repositories.Get(context, orgName, repoName)
 		if err != nil {
-			fmt.Println("Skipped repo", repoName, "because it does not exist or could not be accessed.")
+			fmt.Println(
+				"Skipped repo",
+				repoName,
+				"because it does not exist or could not be accessed.",
+			)
+
 			continue
 		}
 
 		// Get head commit SHA of default branch.
-		defaultBranch, _, err := client.Repositories.GetBranch(context, orgName, repoName, *repo.DefaultBranch, true)
-
+		defaultBranch, _, err := client.Repositories.GetBranch(
+			context,
+			orgName,
+			repoName,
+			*repo.DefaultBranch,
+			true,
+		)
 		if err != nil {
-			fmt.Println("Skipped repo", repoName, "because it's default branch could not be accessed.")
+			fmt.Println(
```

Comment:
why not use Printf :-)

Reply:
Heh. This was part of the original code.
Fixed in a future commit in this PR. :)